docs/modules/user-guide/partials/proc_adding-attributes-to-a-devfile.adoc

```diff
@@ -13,9 +13,9 @@ Devfile attributes can be used to configure various features.
 .Procedure
 
 
-. Define the optional `editorFree` attribute
+. Define a custom attribute
```

Comment:
Update this doc to

Devfile attributes can be used to configure various features and properties according to user's and tool's needs. Attributes are implementation-dependant free-form YAML and can be defined for either of the devfile objects - metadata, components, commands, projects or starterProjects.

...
...

Procedure

1. Define a custom attribute

Consider a user defined use case where an editor is not specified in a devfile, a default is provided. When no editor is needed, use the `editorFree` attribute.

```


schemaVersion: 2.0.0
metadata:
  name: petclinic-dev-environment
  attributes: 
     editorFree: true
components:
  - name: myapp
    kubernetes:
      uri: my-app.yaml
```

--

You may remove the rest of the info from this doc